dxr/templates/nunjucks/breadcrumbs.html

```diff
@@ -0,0 +1,11 @@
+{% macro breadcrumbs(www_root, paths_and_names) %}
+  <div class="breadcrumbs">
+    {%- for path, name in paths_and_names -%}
+      {%- if (not loop.first) or (loop.last and loop.length != 1) -%}
```

Comment:
Sorry for holding you up on something you inherited, but...
1) This is equivalent to `if not loop.first` (since you only get to the `loop.last and loop.length != 1` check if it's `loop.first`, but in that case it's never the case that you're also `loop.last` if `loop.length != 1`).
2) This not/and/or version is the one that came from folder.html - the one that came from file.html was
 `if not loop.first or loop.last`,
 i.e. `if (not loop.first) or loop.last`,
 which *would* add a leading slash if there was only one path term (because the one and only term would be `loop.last`), but we always lead file paths with the tree name (right?), so in our case `loop.last` always implies `not loop.first` for a file (I'm guessing maybe that wasn't always the case?).  So the original file.html version is actually equivalent to `if not loop.first` as well (for us).

So I'm suggesting just making it `if not loop.first`.  :-)